package.json

```diff
@@ -8,11 +8,18 @@
     "@testing-library/user-event": "^13.5.0",
     "echarts": "^5.4.0",
     "echarts-for-react": "^3.0.2",
+    "leaflet": "^1.9.2",
+    "leaflet-geosearch": "^3.7.0",
+    "leaflet.offline": "^3.0.0-rc.1",
+    "localforage": "^1.10.0",
+    "offline-map-react": "^4.4.18",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-leaflet": "^4.1.0",
     "react-router": "^6.4.2",
     "react-router-dom": "^6.4.2",
     "react-scripts": "5.0.1",
+    "react-thermometer-component": "^1.0.1",
```

Comment:
And also remove the react thermometer component dependency as we've decided to gauges from eCharts.